thrift/thrift0.13/src/main/java/com/linecorp/armeria/internal/server/thrift/ThriftDocServicePlugin.java

```diff
@@ -188,18 +188,23 @@ ServiceInfo newServiceInfo(Class<?> serviceClass, Iterable<EndpointInfo> endpoin
     }
 
     @Nullable
-    private MethodInfo newMethodInfo(Method method, Iterable<EndpointInfo> endpoints,
+    private MethodInfo newMethodInfo(Method method, Entry entry,
                                      DocServiceFilter filter) {
         final String methodName = method.getName();
-
         final Class<?> serviceClass = method.getDeclaringClass().getDeclaringClass();
         final String serviceName = serviceClass.getName();
         if (!filter.test(name(), serviceName, methodName)) {
```

Comment:
Question: Shouldn't we use `thriftFunctionName` instead of `methodName` here?

Reply:
Yes.
At first, I thought maybe user would like use the compiled code to filter, this could be more obvious when coding.
But at the example request we use `thriftFunctionName` too, so this one should use `thriftFunctionName`.